build.gradle

```diff
@@ -44,3 +45,24 @@ allprojects {
         }
     }
 }
+
+apply plugin: "build-time-tracker"
+buildtimetracker {
+    reporters {
+        csv {
+            output "build/times.csv"
+            append true
+            header false
```

Comment:
This looks cool! How do you the data?

Reply:
It's associated with builds in the terminal, and afaik a file in the build folder as well.